packages/coil-extension/src/background/services/StreamAssociationsWM2.test.ts

```diff
@@ -0,0 +1,90 @@
+import '@abraham/reflection'
```

Comment:
wonder if we can move this to some jest config, rather than have to repeat it everywhere ??